packages/quantic/force-app/main/default/lwc/quanticRefineModalContent/quanticRefineModalContent.js

```diff
@@ -0,0 +1,219 @@
+import {LightningElement, api} from 'lwc';
+import {getAllFacetsFromStore} from 'c/quanticHeadlessLoader';
+import {
+  initializeWithHeadless,
+  registerComponentForInit,
+} from 'c/quanticHeadlessLoader';
+import filters from '@salesforce/label/c.quantic_Filters';
+import clearAllFilters from '@salesforce/label/c.quantic_ClearAllFilters';
+
+/** @typedef {import("coveo").SearchStatus} SearchStatus */
+/** @typedef {import("coveo").SearchEngine} SearchEngine */
+/** @typedef {import("coveo").BreadcrumbManager} BreadcrumbManager */
+
+const COMMON_FACET_PROPERTIES = ['facetId', 'field', 'label'];
+const DEFAULT_FACET_PROPERTIES = [
+  'numberOfValues',
+  'sortCriteria',
+  'noSearch',
+  'displayValuesAs',
+  'noFilterFacetCount',
+  'injectionDepth',
+];
+const NUMERIC_FACET_PROPERTIES = [
+  'numberOfValues',
+  'sortCriteria',
+  'rangeAlgorithm',
+  'withInput',
+  'formattingFunction',
+];
+const CATEGORY_FACET_PROPERTIES = [
+  'basePath',
+  'noFilterByBasePath',
+  'noFilterFacetCount',
+  'delimitingCharacter',
+  'numberOfValues',
+  'sortCriteria',
+  'withSearch',
+];
+const TIMEFRAME_FACET_PROPERTIES = [
+  'withDatePicker',
+  'noFilterFacetCount',
+  'injectionDepth',
+];
+
+/**
+ * The `QuanticRefineModalContent` component displays a copy of the search interface facets and sort components. This component is intended to be displayed inside a modal to assure the responsiveness when the search interface is displayed on smaller screens.
```

Comment:
instead of "a modal" i think it would be better to directly mention the QuanticModal component you built for this purpose :)